extension/adjustProductsForMapPricing.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function (context, input, cb) {
+  const adjustedProducts = input.products.map((product) => {
+    if (product.flags.isMapProduct) {
+      product.price.unitPrice = null
```

Comment:
We just looked it up in the theme:
We are currently checking if the price object is not null we dont show it.
So we can do `product.price = null`